service/eventer/github/client.go

```diff
@@ -132,10 +132,13 @@ func (e *Eventer) fetchNewDeploymentEvents(project, environment string, etagMap
 		deployments[index].Statuses = deploymentStatuses
 	}
 
-	deployments = e.filterDeploymentsByStatus(deployments)
+	if filterFinished {
+		e.logger.Log("debug", "filtering finished deployment events", "project", project)
+		deployments = e.filterDeploymentsByStatus(deployments)
+	}
```

Comment:
We use `fetchNewDeploymentEvents` for both, fetching the latest deployment events and fetching deployment events continuously. The difference here is though, that we are also interested in the finished ones when fetching the latest deployment events.